frontend/src/cards/CardWrapper.tsx

```diff
@@ -51,29 +80,26 @@ function CardWrapper(props: {
             {!props.hideFooter && (
               <CardContent className={styles.CardFooter}>
                 Sources:{" "}
+                {/* TODO- add commas and "and" between the data sources */}
                 {props.datasetIds.map((datasetId) => (
                   <>
-                    {datasetId === "acs_population-by_race_state_std" && (
-                      <>Population and demographic data from </>
-                    )}
                     <LinkWithStickyParams
                       target="_blank"
                       to={`${DATA_CATALOG_PAGE_LINK}?${DATASET_PRE_FILTERS}=${datasetId}`}
                     >
                       {datasetStore.metadata[datasetId].data_source_name}
                       {". "}
```

Comment:
I know this is going away when you add commas, but I'd just drop the periods now. They're making the links look weird